importer/addProfilePictureUrlToCandidate.js

```diff
@@ -0,0 +1,34 @@
+var Promise = require('bluebird');
+var _ = require('lodash');
+var async = require('async');
+var Candidates = require('../models/candidate');
+
+//Mongoose
+var mongoose = require('mongoose');
+mongoose.Promise = Promise;
+mongoose.connect('mongodb://localhost:27017/dc-campaign-finance', {
+    server: {
+        socketOptions: {
+            keepAlive: 5,
+            connectTimeoutMS: 30000
+        }
+    }
+});
+
+Candidates.find()
```

Comment:
I would make sure to run this locally and make sure you don't run into any memory issues when processing all the candidates.  Since there are not that many I would not expect this to be an issue.